go-controller/pkg/config/config.go

```diff
@@ -533,6 +534,9 @@ type UserDefinedNetworksConfig struct {
 	// ConntrackMarkBase will be the base to calculate the
 	// ct_mark for a user defined network, that's ct_mark=base+network-id
 	ConntrackMarkBase uint `gcfg:"ct-mark-base"`
+	// VRFTableBase will be the base to calculate the
+	// vrf table id for a user defined network, that's table=base+network-id
+	VRFTableBase uint `gcfg:"user-defined-network-vrf-table-base"`
```

Comment:
I am planning on removing the VRF tableID calculation all together as @pperiyasamy suggested we can calculated it from the VRF device (one per network) index + base.

There is already code doing that for egress IP

https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/node/controllers/egressip/egressip.go#L1323

About VRF device name since we are going to have one mp port per network nameing it something like "vrf-"+mp will be enough.